libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -115,6 +125,13 @@ void AC_PrecLand::init()
     _backend = nullptr;
     _backend_state.healthy = false;
 
+    // Create inertial history buffer
+    check_inertial_buffer();
+    /* NOTE: If this check below returns without creating buffer and backend then SITL crashes */
+    if (_inertial_history == nullptr) {
+        return; // allocation failed
```

Comment:
great, comment removed.  Actually I'll make check_inertial_buffer() return true/false on successful/failure.

Reply:
Just to note: when I was doing my initial testing in SITL and this returned without doing anything it would sometimes crash the SITL system hard.  This was a note for future people like your good selves who actually know what you're talking about to take a closer look and make sure there isn't some flight-ending bug in there that could be resolved.

Reply:
@fnoop, yes, I found and reproduced the issue.  It was because update() was trying to access the buffer but it was a nullptr.  It's fixed now and I've retested that SITL doesn't crash anymore.